.ci.yaml

```diff
@@ -4620,33 +4620,45 @@ targets:
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_tool_startup__macos
 
-  - name: Linux flutter_packaging_beta
+  - name: Linux flutter_packaging_beta_linux
     recipe: packaging/packaging
     scheduler: release
     bringup: true
     enabled_branches:
       - beta
+    properties:
+      task_name: flutter_packaging_beta_linux
+
 
-  - name: Mac flutter_packaging_beta
+  - name: Mac flutter_packaging_beta_mac
     recipe: packaging/packaging
     scheduler: release
     bringup: true
     enabled_branches:
       - beta
+    properties:
+      task_name: flutter_packaging_beta_mac
+
 
-  - name: Mac_arm64 flutter_packaging_beta
+  - name: Mac_arm64 flutter_packaging_beta_mac_arm64
```

Comment:
Do we need the platform_architecture at the end, we can probably pass a cpu dimension like in here: https://cs.opensource.google/flutter/flutter/+/master:.ci.yaml;l=2023?q=drone%20dimensions&ss=flutter&start=21

Reply:
Let's remove the _linux and _mac at the end of the names

Reply:
Removed

Reply:
Thank you!